buildSrc/src/main/kotlin/net/skyscanner/backpack/tokens/Case.kt

```diff
@@ -0,0 +1,23 @@
+/**
+ * Backpack for Android - Skyscanner's Design System
+ *
+ * Copyright 2018-2021 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package net.skyscanner.backpack.tokens
+
+import com.google.common.base.CaseFormat
+
+fun String.changeCase(from: CaseFormat, to: CaseFormat): String =
```

Comment:
This seems to be a simple wrapper. Is it necessary? Do you think it will grow into something else ?